assets/src/components/map.tsx

```diff
@@ -336,24 +316,79 @@ const AutoCenterMapOn = ({
     // `autopanstart` is invoked when opening a popup causes the map to pan to fit it
     autopanstart: () => setShouldAutoCenter(false),
   })
+}
 
+export type InterruptibleFollowerProps = InteractiveFollowState & FollowerProps
+
+// Component which provides following capability and configures the map to stop
+// the follower component when a user interacts with the map
+export const InterruptibleFollower = ({
+  shouldFollow,
+  setShouldFollow,
+  isUpdating,
+  positions,
+  onUpdate,
+}: InterruptibleFollowerProps) => {
+  useStopFollowingOnInteraction(isUpdating, setShouldFollow)
+
+  const turnOnFollowing = useCallback(
+    () => setShouldFollow(true),
+    [setShouldFollow]
+  )
```

Comment:
question (non-blocking): Why `useCallback` here? I'm guessing it prevents re-renders, but I'm not too familiar with this hook

Reply:
I did it to prevent rerenders, but I don't actually remember if it was needed. It seemed like it was causing issues but it might have just been something I did while debugging and didn't reevaluate.

Reply:
So this `useCallback` doesn't actually seem to do anything? so I'll remove it!

Reply:
[`efb9a92` (#1891)](https://github.com/mbta/skate/pull/1891/commits/efb9a92e7648a720e61c66030b73133176ffe0d2)